Kickstarter-iOS/Views/Controllers/CancelPledgeViewController.swift

```diff
@@ -4,18 +4,30 @@ import Library
 import Prelude
 import UIKit
 
+protocol CancelPledgeViewControllerDelegate: AnyObject {
+  func cancelPledgeViewController(
+    _ viewController: CancelPledgeViewController,
+    didCancelPledgeWith message: String
+  )
```

Comment:
I think in this case we'd want like:
```swift
func cancelPledgeViewController(
  _ viewController: CancelPledgeViewController,
  didCancelPledgeWithMessage message: String
)
```
Just because it isn't obvious at the callsite what `String` is.